### DISCUSSION TITLE
[CPP] 如何合适的调试debug?

### DISCUSSION THREAD

**Discussion by JigaoLuo** [j178/leetgo#301] [Q&A] [answered]
CPP下，`leetgo test last -L`之后编译的二进制是在哪儿呢？想找到它 直接gdb这个二进制

**Comment by j178** [comment, marked as answer]
可以设置一下环境变量 `DEBUG=1`, 然后运行 `leetgo test last -L` 可以看到这样的输出：
```
2024/05/25 01:25:28 INFO building cmd="E:\Scoop\apps\gcc\current\bin\g++.exe -O2 -std=c++17 -I D:\Projects\goland\leetgo\out\cpp -o C:\Users\jo\AppData\Local\Temp\leetgo\two-sum-cpp.exec D:\Projects\goland\leetgo\out\cpp\0001.two-sum\solution.cpp"
```
这个就是 leetgo 编译 cpp 时候的命令，二进制是放在 `C:\Users\jo\AppData\Local\Temp\leetgo\two-sum-cpp.exec` 下的，你也可以自己用 gcc/clang 编译，输出到任意指定的位置。

**Comment by JigaoLuo** [comment]
谢谢 
Linux下的二进制在 `/tmp/leetgo/two-sum-cpp.exec `
